pages/coq-team.html

```diff
@@ -0,0 +1,503 @@
+    <div class="position">Inria Researcher in the Pesto Team, Nancy, France</div>
+    <div class="role">RM: 8.10</div>
+    <div class="components">Nix, micromega</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/assia_mahboubi.jpg" alt="Assia Mahboubi" width="200px"/></div>
+    <div class="name"><a href="http://people.rennes.inria.fr/Assia.Mahboubi/">Assia Mahboubi</a></div><div class="nickname">amahboubi</div>
+    <div class="position">Inria Researcher in the Gallinette Team, Nantes, France</div>
+    <div class="role"><a href="https://cordis.europa.eu/project/id/101001995">ERC FRESCO PI</a></div>
+    <div class="components">Ring, Mathematical Components</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/erikmd.png" alt="Érik Martin-Dorel" width="170px"/>
+    </div>
+    <div class="name"><a href="https://www.irit.fr/~Erik.Martin-Dorel/">Érik Martin-Dorel</a></div><div class="nickname">erikmd</div>
+    <div class="position">Associate Professor at IRIT, Toulouse University, France</div>
+    <div class="components">Primitive Types, Docker, Proof-General</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/palmskog.jpg" alt="Karl Palmskog" width="190px"/>
+    </div>
+    <div class="name"><a href="https://setoid.com/">Karl Palmskog</a></div><div class="nickname">palmskog</div>
+    <div class="position">Researcher at the KTH Royal Institute of Technology, Stockholm, Sweden</div>
+    <div class="role">Coq-community manager</div>
+    <div class="components">coq-community, opam, website, Roosterize</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/cpitclaudel.jpg" alt="Clément Pit-Claudel" width="170px"/>
+    </div>
+    <div class="name"><a href="http://pit-claudel.fr/clement/">Clément Pit-Claudel</a></div><div class="nickname">cpitclaudel</div>
+    <div class="position">PhD Student at MIT, MA, USA</div>
+    <div class="components">Documentation, Alectryon, Proof-General, Company-Coq</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/proux.jpg" alt="Pierre Roux" width="200px"/>
+    </div>
+    <div class="name"><a href="https://www.onera.fr/fr/staff/pierre-roux">Pierre Roux</a></div><div class="nickname">proux</div>
+    <div class="position">Researcher at Onera, Toulouse, France</div>
+    <div class="components">Primitive Types</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/pi8027.jpg" alt="Kazuhiko Sakaguchi" width="200px"/></div>
+    <div class="name">Kazuhiko Sakaguchi</div><div class="nickname">pi8027</div>
+    <div class="position">PhD Student at the University of Tsukuba, Japan</div>
+    <div class="components">Coercions, Extraction, Hierarchy Builder</div>
```

Comment:
Maybe also "Mathematical Components"? I'm not an ssreflect-maintainer though.

Reply:
Indeed!